### DISCUSSION TITLE
Height setting

### DISCUSSION THREAD

**Discussion by FreedomClannad** [epam/miew#416] [Q&A]
I use Vue environment to import this project. I display 24px at the height of Vue environment. I follow the API in examples JS, API Html is displayed normally. Why can't the height of my project be displayed normally. There are also detailed API interface documents.

demo：https://stackblitz.com/edit/vue-2ho19c?file=src%2Fmain.js
Upper right corner Open in New Window You can see the effect

**Comment by AnaMoskvina** [comment]
Dear @FreedomClannad,

Thank you for your question. 
The behavior you described is actually a bug, and we will fix in the near future.
As a workaround, to make the viewer occupy 100% height of miew container, please, add following rule to your styles:
`.miew-canvas {
  height: 100%;
}`

Feel free to contact us, if you have any questions, we will be happy to help.

Best regards,
Miew team

**Comment by FreedomClannad** [comment]
@AnaMoskvina Thank you for your help. I would like to ask if there are more specific api documents. I see that there are relatively few examples, and I don't know which interfaces are there.

**Comment by AnaMoskvina** [comment]
Dear @FreedomClannad,
Thank you for your question again.
Did you have a chance to check out the documentation?
If no - to do so, please, clone/copy the project to your machine, and open documentation/index.html 
You will see this page. Here you can find an extensive API description.
![docs](https://user-images.githubusercontent.com/77681522/154436063-16304b41-81dc-49fc-b325-b4bc995c6a98.PNG)

Best regards,
Miew team

**Comment by FreedomClannad** [comment]
thank you very much for your help
